Tests/EventDispatcherTests.swift

```diff
@@ -0,0 +1,198 @@
+import Quick
+import Nimble
+import Clappr
+
+class EventDispatcherTests: QuickSpec {
+
+    override func spec() {
+        describe("BaseObject") {
+
+            var baseObject: EventDispatcher!
```

Comment:
Since this is not a `BaseObject` we can change the var name to `eventDispatcher`.